package/requirements.txt

```diff
@@ -9,7 +9,7 @@ watchgod~=0.8.2
 plotly>=4.0
 pandas>=0.24
 sqlalchemy>=1.4, <3
-strawberry-graphql>=0.99.0, <1.0
+strawberry-graphql>=0.191.0, <1.0
```

Comment:
`0.191` doesn't exist right? https://pypi.org/project/strawberry-graphql/#history

Reply:
updated to 0.192